v3/src/components/graph/models/graph-layout.ts

```diff
@@ -124,27 +124,74 @@ export class GraphLayout extends DataDisplayLayout implements IAxisLayout {
   @override get computedBounds() {
     const {desiredExtents, tileWidth, tileHeight} = this,
       topAxisHeight = desiredExtents.get('top') ?? 0,
+      bannersHeight = desiredExtents.get('banners') ?? 0,
       leftAxisWidth = desiredExtents.get('left') ?? 20,
       bottomAxisHeight = desiredExtents.get('bottom') ?? 20,
       legendHeight = desiredExtents.get('legend') ?? 0,
       v2AxisWidth = desiredExtents.get('rightNumeric') ?? 0,
       rightAxisWidth = desiredExtents.get('rightCat') ?? 0,
       plotWidth = tileWidth - leftAxisWidth - v2AxisWidth - rightAxisWidth,
-      plotHeight = tileHeight - topAxisHeight - bottomAxisHeight - legendHeight,
-      newBounds: Record<GraphPlace, Bounds> = {
-        left: {left: 0, top: topAxisHeight, width: leftAxisWidth, height: plotHeight},
-        top: {left: leftAxisWidth, top: 0, width: tileWidth - leftAxisWidth - rightAxisWidth, height: topAxisHeight},
-        plot: {left: leftAxisWidth, top: topAxisHeight, width: plotWidth, height: plotHeight},
-        bottom: {left: leftAxisWidth, top: topAxisHeight + plotHeight, width: plotWidth, height: bottomAxisHeight},
-        legend: {left: 6, top: tileHeight - legendHeight, width: tileWidth - 6, height: legendHeight},
-        rightNumeric: {left: leftAxisWidth + plotWidth, top: topAxisHeight, width: v2AxisWidth, height: plotHeight},
-        rightCat: {left: leftAxisWidth + plotWidth, top: topAxisHeight, width: rightAxisWidth, height: plotHeight},
-        yPlus: {left: 0, top: topAxisHeight, width: leftAxisWidth, height: plotHeight} // This value is not used
+      plotHeight = tileHeight - bannersHeight - topAxisHeight - bottomAxisHeight - legendHeight,
+      newBounds: Record<GraphExtentsPlace, Bounds> = {
```

Comment:
With the introduction of `bannersHeight` some lines in the definition of `newBounds` became too long according to the linting rules. Breaking up just those particular lines into two lines didn't help with legibility, in my opinion. Ultimately, it seemed to me that the below made the various calculations easier to grok. If there's a good reason we should try to minimize the line breaks, though, I can bring it back more inline with the previous state.